pages/partials/externalGradingLiveUpdate.ejs

```diff
@@ -19,8 +19,8 @@
 
       var status = $(elem).data('grading-job-status');
       var submissionId = $(elem).attr('id').replace('submission-', '');
-      updateStatus(submissionId, status);
```

Comment:
I believe this call is still necessary to show the correct status immediately after page load, though I do see that it's clearly being called with incorrect parameters. Maybe this would work instead?

```js
updateStatus({ id: submissionId, grading_job_status: status });
```

Reply:
Note that this function does not retrieve the new status, it only uses the status from the data attributes. So presumably the data being updated here was already used in rendering. It would be updated in the init event, later. So this call is not actually necessary.

Reply:
We don't actually render the status messages or spinners on the server; that's done strictly client-side. We could make the change to render them on the server if you want.

Reply:
Of course, things were working well enough before with the obviously incorrect call, so I think it would be ok to delete it. But my intention in writing this was definitely to ensure that we can show something useful even before we've gotten the initial state back from the websocket.

Reply:
Things were working because, if this display was expected to be shown, then grading is also pending, so the socket would be immediately initialized and the status would be updated. So this call, if it were correct, would only serve to set the status between when the page is loaded and when the socket returned the new status. This time is too short to be noticed in the vast majority of cases. In any case, I've added it back with your suggestion.